### PR TITLE
Remove cachedb

### PR DESCRIPTION
**Pull request by superkhau**
- Remove `_cache.db` dir and its contents
- Add `config.json` to `.gitignore`

### CODE REVIEW

**Comment by superkhau** [comment]
@rmg Review please.

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ node_modules
 .auth.json
 _cache.db
 all*.json
+config.json
```
Comment (rmg) [inline .gitignore]: Not sure why this is necessary, or why you would even create this file. You can run against the configs in-place: `./bin/sync.js projects/nodeops.json`

**Comment by rmg** [comment]
Not sure why we need to ignore config.json, but I don't see any harm in it.

LGTM.

File: .gitignore (hunk 1 above)
Comment (superkhau) [inline .gitignore]: Should be in another PR, but got lazy. Didn't want to include my local `config.json` that was already in my local clone.

Comment (superkhau) [inline .gitignore]: I run `bin/sync` config.json` like it says in the README for example repos with settings that are different than `projects/loopback.json` is using. I should probably merge them together at some point.

Comment (rmg) [inline .gitignore]: It's fine in this PR, you at least made it a separate commit :-)
